sdk/documentintelligence/ai-document-intelligence-rest/src/documentIntelligence.ts

```diff
@@ -40,7 +41,7 @@ export default function createClient(
       logger: options.loggingOptions?.logger ?? logger.info,
     },
     credentials: {
-      scopes: options.credentials?.scopes ?? ["https://cognitiveservices.azure.com/.default"],
+      scopes: options.credentials?.scopes ?? [KnownDocumentIntelligenceAudience.AzurePublicCloud],
```

Comment:
Either we could detect and add /.default if it is missing? or users will have to add `/.default` to known audience by themselves.

Reply:
I would want such a logic to be in core ideally

Reply:
Core has the default setting to `${options.endpoint}/.default` if scope is absent, but I think this would not be applied to data-plane, right? because the endpoint and scope is different.

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/core-client/src/serviceClient.ts#L250-L252